### ISSUE TITLE
Submodule `breeze-icons` missing

### ISSUE THREAD

**Issue by tinxx** [bearsh/origami#2]
You define a dependency to `breeze-icons` here:
https://github.com/bearsh/origami/blob/master/icons.qrc

But you never pull them in as a submodule. One has to manually clone [breeze-icons](https://github.com/KDE/breeze-icons) into kirigami dir.

**Comment by bearsh** [comment]
yes that's right but if I want to include breeze-icons as a submodule, it has to be a submodule of kirigami (as far as I understand git submodule) and to do so I would need to fork kirigami.

maybe that's the way to go considering the cmake build system of kirigami clones breeze-icons. and I far as I know there may be good reasons to do so instead integrate it as submodule.

I'm leaving it open until I've talked to the kirigami devs and came to a conclusion.
